lib/backend/k8s/k8s_fv_test.go

```diff
@@ -747,7 +747,7 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 	}()
 
 	It("should not error on unsupported List() calls", func() {
-		objs, err := c.List(model.BGPPeerListOptions{})
+		objs, err := c.List(model.NodeBGPPeerListOptions{})
```

Comment:
We might want to change this to something else given we're going to be supporting this soon!

Not sure if you want to do it in this PR, or if you'll remember in your other ones, but it might be that this test passes even though the resource is actually supported given the assertions it's making.